engine/src/main/java/org/hibernate/search/backend/DeleteByQueryLuceneWork.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend;
+
+import java.io.Serializable;
+
+import org.hibernate.search.backend.impl.WorkVisitor;
+
+/**
+ * Representation of deleteByQuery(...) from Lucene. Currently not all functionality of Lucene is supported but can be
+ * added
+ *
+ * @author Martin Braun
+ */
+public class DeleteByQueryLuceneWork extends LuceneWork implements Serializable {
+
+	private static final long serialVersionUID = -5384359572337884952L;
```

Comment:
I know the IDE recommends having these.. but there are actually good reasons to not specify a serialVersionUID.
We decided to never decplare them.